images/coog/py3/Dockerfile

```diff
@@ -1,24 +1,18 @@
-FROM python:3-slim
+FROM alpine:edge
 MAINTAINER Coopengo <support@coopengo.com>
 
-RUN apt-get update && apt-get install -y --no-install-recommends \
-        gcc \
-        vim \
-        libpq-dev \
-        libldap2-dev \
-        libsasl2-dev \
-        redis-tools \
-        graphviz \
-        cifs-utils \
-    && apt-get clean && rm -rf /var/lib/apt/lists/*
-
-RUN pip install \
+RUN echo "http://dl-cdn.alpinelinux.org/alpine/edge/testing" >> /etc/apk/repositories \
```

Comment:
stable

Reply:
We will wait until alpine-3.8
Mais au moins les tests sont fait